app/src/js/actions/index.js

```diff
@@ -366,6 +366,47 @@ export const updateInputs = (questionId, payload) => ({
   }
 });
 
+export const listInputs = (options) => {
```

Comment:
There's not actually an endpoint for this in the API, so we'll need to eventually add this. Non-breaking though, so this won't block this PR.

Reply:
I see now. It's using the default data lambdas findById and findAll